index.js

```diff
@@ -80,14 +78,69 @@ Lingo.prototype.fetchKitOutline = function (id, version) {
  * @param {integer} page the page of items
  * @returns {Promise} Success the section and the items matching the page/limit
  */
-Lingo.prototype.fetchSection = function (id, version, page = 1, limit = 50) {
-    let path = `/sections/${id}`
-    let v = version
-    let params = { qs: { v, page, limit } }
-    return this.call('GET', path, params).then(res => {
-        return res.section
-    })
-}
+Lingo.prototype.fetchSection = function(id, version, page = 1, limit = 50) {
+  let path = `/sections/${id}`;
+  let v = version;
+  let params = { qs: { v, page, limit } };
+  return this.call("GET", path, params).then(res => {
+    return res.section;
+  });
+};
+
+/**
+ * Fetch a section and optionally page through items with it
+ * @param {uuid} id the section uuid
+ * @param {integer} version the version number of the section to fetch
+ * @param {integer} limit The max number of items to fetch
+ * @param {integer} page the page of items
+ * @returns {Promise} Success the section and the items matching the page/limit
+ */
+Lingo.prototype.fetchAssetsForHeading = function(
+  sectionId,
+  headingId,
+  version
+) {
+  return new Promise((resolve, reject) => {
+    let page = 1;
+    let found = false;
+    const results = [];
+
+    function isMatch(item) {
+      return (
+        item.type == "heading" &&
+        (item.data.content == headingId || item.uuid == headingId)
```

Comment:
It is recommended to use `===` for equality checks in JS because otherwise it will do type coercion, which can get weird. Unless you have a specific reason you want to it coerce values to a comparable type.